Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -683,6 +683,43 @@ final class UIViewExtensionsTests: XCTestCase { // swiftlint:disable:this type_b
         view.removeBlur()
         XCTAssertFalse(view.subviews.first is UIVisualEffectView)
     }
+    
+    func testSeparatorAddsSubview() {
```

Comment:
The standard for this project is that each extension function has only 1 test function.